platform_versioned_docs/version-24.1.0/credentials/managed_identities.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Managed identities"
+description: "Configure managed identities in Seqera Platform."
+date: "12 Jun 2024"
+tags: [organizations, administration, managed identities, clusters, ssh, credentials]
+---
+
+With traditional SSH credentials, all workspace users access HPC clusters with the same set of credentials configured in the workspace. While this simplifies access control to computing resources, all user activity on your HPC cluster is logged under the same user credentials. This also means that all Seqera workspace users have the same access permissions on your HPC cluster. When multiple workspace users run pipelines and consume compute resources, all activity is reflected under the same user on your HPC cluster. 
```

Comment:
A suggestion to enhance these paragraphs adding more context why this is good ( audit, security, fair usage,)
feel free to edit/enhance it at will
Managed identities offer significant advantages for High-Performance Computing (HPC) environments by enabling granular access control for individual workspace users. Unlike traditional SSH credentials, which allow all workspace users to access HPC clusters using the same set of credentials, managed identities ensure each user’s activity is logged under their own credentials. This preservation of user identity is crucial as it naturally inherits the HPC system's fair usage queue policies, mitigates the noisy neighbor problem, and reduces the long wait times associated with First-In-First-Out (FIFO) queues common with shared SSH credentials.

Traditional SSH credentials, while simplifying access control to computing resources, result in all user activities on the HPC cluster being logged under the same user credentials. This means all Seqera workspace users have the same access permissions on your HPC cluster, leading to indistinguishable user activities. When multiple users run pipelines and consume compute resources, their activities are recorded under a single user identity, making it challenging to trace individual actions and manage resource allocation efficiently.

Managed identities resolve these limitations by allowing administrators to configure a managed identity at the organizational level for access to supported HPC compute environments. This managed identity is selected for authentication similarly to traditional credentials but contains multiple user credentials, each tied to a unique Seqera user. This setup not only preserves the identity of the user launching workflows on the HPC cluster or compute environment but also improves traceability and adherence to data access policies.

Moreover, with managed identities, users only have the access permissions that their system administrators have granted, minimizing the risk of unauthorized read/write operations in restricted folders. In contrast, shared SSH credentials provide all workspace users with the same access level on the HPC side, which is often more extensive than what an individual user would typically have. By grouping individual user SSH credentials into a single element, managed identities allow administrators to streamline user login and compute environment access while maintaining visibility into data access and compute resource usage for each user.

Reply:
Someone has been playing with ChatGPT ;)